RPC-methods.md

```diff
@@ -20,4 +20,18 @@ Resolve types that are currently supported:
 ### Return
 | Return Type | Description                                              |
 | :---------- | :------------------------------------------------------- |
-| `str`       | The full path of where the `fileName` should be located. |
\ No newline at end of file
+| `str`       | The full path of where the `fileName` should be located. |
+
+## textDocument/get_page_ast
+Given a .txt file, return the abstract syntax tree (AST) of the page that is created from parsing that file.
+
+### Parameters
+| Parameter           | Description                                              |
```

Comment:
Just updated the PR so that we don't need the text at all. You pointing this out made me realize there was no need for it (thanks!).

Right now, the path is required but a future implementation can potentially use the `fileId` of the file we want instead of the full path, if it is known.